sensu/files/plugins/graphite-data.rb

```diff
@@ -43,16 +43,16 @@ class CheckGraphiteData < Sensu::Plugin::Check::CLI
     :required => false
 
   option :warning,
-    :description => 'Generate warning if given value is above received value',
+    :description => 'Generate warning if given value is above received value. Or comma seperated range.',
     :short => '-w VALUE',
     :long => '--warn VALUE',
-    :proc => proc{|arg| arg.to_f }
```

Comment:
It does! Yay magic. Though it will need testing if it works on the version of ruby that suensu ships… I tested it in `irb` but I'm not sure if it works for sensu's ruby.